week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
+                <div id="humanscore">0</div>
+                <div id="vs"> : </div>
+                <div id="computerscore">0</div>
+            </div>
+            <div class="text">버튼을 눌러 시작하세요!</div>
+        </div>
+        
+        <div class="middle">
+            <div id="human_select"><img id="img" src="./img/paper.png"/></div>
+            <div id="robot_select"><img id="robotimg" src="./img/paper.png"/></div>
+        </div>
+        <div class="bottom">
```

Comment:
class 이름은 목적과 의미가 드러나도록 명확하게 짓는 것이 좋습니다.
```suggestion
        <div class="option">
```